packages/qwik-auth/src/index.ts

```diff
@@ -141,7 +141,7 @@ async function authAction(
     skipCSRFCheck,
   });
   res.headers.forEach((value, key) => {
-    if (!req.headers.has(key)) {
+    if (!req.headers.has(key) || key === 'set-cookie') {
```

Comment:
Can we add a comment explaining the reasoning for this....

Reply:
Added a comment, if it doesn't include the details you feel is important feel free to suggest another :).